src/Sentry/Extensibility/DisabledHub.cs

```diff
@@ -55,10 +56,11 @@ public void WithScope(Action<Scope> scopeCallback)
         /// <summary>
         /// Returns a dummy transaction.
         /// </summary>
-        public ITransaction StartTransaction(string name, string operation) => new Transaction(this, name, operation)
-        {
-            IsSampled = false
-        };
+        public ITransaction StartTransaction(
+            ITransactionContext context,
+            IReadOnlyDictionary<string, object?> customSamplingContext) =>
```

Comment:
Decided not to roll a custom type for this and just use `IReadOnlyDictionary<string, object?>` for custom sampling context. Thoughts @bruno-garcia?